src/algoesup/magics.py

```diff
@@ -178,6 +184,13 @@ def ruff(line: str) -> None:
         ```
     """
     args = parse_argstring(ruff, line)
+    if args.jupyter:
+        # See https://docs.astral.sh/ruff/rules/ for more information on codes.
+        ruff_args = [
+            "--select", "A", "B", "C90", "D", "E", "W", "F", "N", "PL",
```

Comment:
Won't this get space-separated when calling ruff, i.e. `ruff --select A B ...`? That will lead to errors because ruff will think B is a file to lint. It has to be `ruff --select A,B,C90,...`.

Reply:
There is something else to take note of from this mistake. `I don't think %ruff` or possibly `subprocess.run()` are passing basic errors up the chain, which is a bit of a problem.